### DISCUSSION TITLE
GitHub Pro是按月付费的还是买断的？

### DISCUSSION THREAD

**Discussion by SqrtTan** [Meloong-Git/PCL#3435] [提问] [answered]
我想开个Pro，看看是月制还是买断（

**Comment by Deep-Dark-Forest** [comment]
月付

**Comment by zkitefly** [comment, marked as answer]
![image](https://github.com/Hex-Dragon/PCL2/assets/64117916/528b445d-eace-4212-bd27-eac888ebd68e)
月付或年付

**Comment by SqrtTan** [comment]
好吧，想想也是，4美元买断确实不太可能……

**Reply by Deep-Dark-Forest** [reply]
如果是买断的话账号交易会非常严重，巨婴亏死

**Comment by xxtg666** [comment]
可以弄学生包，有效期内免费给你pro（我就是

**Reply by SqrtTan** [reply]
我们学校还没造好……现在是在别人学校的实验室里上学的……~所以我连学校填哪个名字都不知道~
本人初生

**Reply by wuliaodexiaoluo** [reply]
要求至少有 xxx@xxx.edu.cn 或者其他可以被 GitHub 认为是高等学校提供的电子邮件地址
初中应该不行，因为学校没有提供电子邮件的能力，描述也没有初级学校的选项

**Reply by ChouChiu** [reply]
> 要求至少有 [xxx@xxx.edu.cn](mailto:xxx@xxx.edu.cn) 或者其他可以被 GitHub 认为是高等学校提供的电子邮件地址 初中应该不行，因为学校没有提供电子邮件的能力，描述也没有初级学校的选项

初中其实可以，只需要有一个教育邮箱就行，参考 https://frcss.edu.hk

**Reply by LinQingYuu** [reply]
> > 要求至少有 [xxx@xxx.edu.cn](mailto:xxx@xxx.edu.cn) 或者其他可以被 GitHub 认为是高等学校提供的电子邮件地址 初中应该不行，因为学校没有提供电子邮件的能力，描述也没有初级学校的选项
> 
> 初中其实可以，只需要有一个教育邮箱就行，参考 https://frcss.edu.hk

境内高中基本没有这个东西
